### PR TITLE
1.7 exercises

### PR DESCRIPTION
**Pull request by ravikumar89**
This is a test pull request to check with omnicode.

[Review on Reviewable](https://reviewable.io/reviews/ravikumar89/valarmorghulis/1)

### CODE REVIEW

**Comment by ravikumar89** [comment]
JB adnkn sankcnksqc dqj

**Comment by ravikumar89** [comment]
this is a sample comment to check how its working.

**Comment by ravikumar89** [comment]
hello world

**Comment by ravikumar89** [comment]
this is a new comment

**Comment by ravikumar89** [comment]
subcomment

**Comment by ravikumar89** [comment]
smot cg ggvher xyggvxhin

**Comment by ravikumar89** [comment]
vjhbhjngvybh fyghhbhbvhhbh

**Comment by jayshreeanand** [comment]
This is comment on a pull request

**Comment by jayshreeanand** [comment]
@ravikumar89 This is a reply on a pull request

**Comment by ravikumar89** [comment]
@jayshreeanand: test

File: config/routes.rb [hunk 1]
```diff
@@ -55,4 +55,5 @@
   #   end
```
Comment (jayshreeanand) [inline config/routes.rb]: test

File: config/routes.rb [hunk 2]
```diff
@@ -55,4 +55,5 @@
...
 
   root 'myapplication#hello'
+  #root 'myapplication#goodbye'
 end
```
Comment (jayshreeanand) [inline config/routes.rb]: test comment position58

**Comment by ravikumar89** [comment]
@jayshreeanand: TEst message

File: app/controllers/myapplication_controller.rb [hunk 3]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
 	protect_from_forgery with: :exception
 
 	def hello
-		render text: "hello world! Hai How are you?"
+		render text: "¡hola, mundo!"
+	end
+
+	def goodbye
```
Comment (jayshreeanand) [inline app/controllers/myapplication_controller.rb]: test comment on app controller

File: app/controllers/myapplication_controller.rb [hunk 4]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
 	protect_from_forgery with: :exception
 
 	def hello
-		render text: "hello world! Hai How are you?"
```
Comment (jayshreeanand) [inline app/controllers/myapplication_controller.rb]: test reply on comment

Comment (jayshreeanand) [inline app/controllers/myapplication_controller.rb]: test reply

File: app/controllers/myapplication_controller.rb [hunk 5]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
...
+		render text: "¡hola, mundo!"
+	end
+
+	def goodbye
+		render text: "goodbye, world!"
```
Comment (jayshreeanand) [inline app/controllers/myapplication_controller.rb]: test comment tuesday

File: config/routes.rb [hunk 6]
```diff
@@ -55,4 +55,5 @@
   #   end
 
   root 'myapplication#hello'
+  #root 'myapplication#goodbye'
```
Comment (jayshreeanand) [inline config/routes.rb]: tet test test

File: app/controllers/myapplication_controller.rb (hunk 4 above)
Comment (jayshreeanand) [inline app/controllers/myapplication_controller.rb]: Test from console

Comment (jayshreeanand) [inline app/controllers/myapplication_controller.rb]: Test from console

File: config/routes.rb (hunk 6 above)
Comment (jayshreeanand) [inline config/routes.rb]: Hello work

Comment (ravikumar89) [inline config/routes.rb]: Best wishes

Comment (ravikumar89) [inline config/routes.rb]: @ravikumar89 what ?

File: app/controllers/myapplication_controller.rb (hunk 4 above)
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: @jayshreeanand mobile

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Hello

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: @ravikumar89 hi

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Test

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: @ravikumar89 hi

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: @ravikumar89 hi1

File: config/routes.rb (hunk 6 above)
Comment (ravikumar89) [inline config/routes.rb]: @jayshreeanand test

Comment (ravikumar89) [inline config/routes.rb]: Another test

File: app/controllers/myapplication_controller.rb [hunk 7]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
 	protect_from_forgery with: :exception
 
 	def hello
-		render text: "hello world! Hai How are you?"
+		render text: "¡hola, mundo!"
+	end
```
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Benson

File: app/controllers/myapplication_controller.rb [hunk 8]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
 	protect_from_forgery with: :exception
```
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Forgery ?

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: @ravikumar89 what?

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Notjing

File: app/controllers/myapplication_controller.rb (hunk 4 above)
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Help

File: app/controllers/myapplication_controller.rb [hunk 9]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
...
```
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Remove the blank line

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Test

File: app/controllers/myapplication_controller.rb (hunk 4 above)
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Test

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Hi

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Test

Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Hello

File: app/controllers/myapplication_controller.rb (hunk 8 above)
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: Wtf

File: config/routes.rb [hunk 10]
```diff
@@ -55,4 +55,5 @@
   #   end
 
   root 'myapplication#hello'
```
Comment (ravikumar89) [inline config/routes.rb]: Unmodified

File: app/controllers/myapplication_controller.rb [hunk 11]
```diff
@@ -4,6 +4,10 @@ class MyapplicationController < ActionController::Base
...
 
 	def hello
-		render text: "hello world! Hai How are you?"
+		render text: "¡hola, mundo!"
+	end
+
+	def goodbye
+		render text: "goodbye, world!"
 	end
 end
```
Comment (ravikumar89) [inline app/controllers/myapplication_controller.rb]: End

**Pull request closed without merging**